src/gui/layout/qgslayoutelevationprofilewidget.cpp

```diff
@@ -431,6 +475,80 @@ QgsExpressionContext QgsLayoutElevationProfileWidget::createExpressionContext()
   return mProfile->createExpressionContext();
 }
 
+void QgsLayoutElevationProfileWidget::setDesignerInterface( QgsLayoutDesignerInterface *iface )
+{
+  mInterface = iface;
+  QgsLayoutItemBaseWidget::setDesignerInterface( iface );
+}
+
+void QgsLayoutElevationProfileWidget::copySettingsFromProfileCanvas( QgsElevationProfileCanvas *canvas )
```

Comment:
Would it be worth adding a test for this function?

Reply:
I'll defer till after freeze :wink: